src/utils.ts

```diff
@@ -211,3 +213,15 @@ export const handleRetries = async (
 
   return { retried: false };
 };
+
+/**
+ * Keys need to expiry to keep the redis clean. Some flows are usually faster,
+ * so it's okay to expiry its keys after a shorter amount of time.
+ */
+export const getKeyExpirySec = (flowName: string) => {
+  if (flowName.startsWith("access")) {
+    return ACCESS_FLOW_KEY_EXPIRY_SEC;
+  }
+
+  return DEFAULT_KEY_EXPIRY_SEC;
+};
```

Comment:
```suggestion
export const getKeyExpiration = (flowName: string) => return flowName.startsWith("access") ? ACCESS_FLOW_KEY_EXPIRY_SEC : DEFAULT_KEY_EXPIRY_SEC;
```

Minor, but I'd prefer this snippet for this specific scenario. Might change later on tho. (I dont like the getKeyExpiry sec either but might be just me, and understand the convention based on the static-s)

Reply:
Attention! If you commit this, we should change the method name everywhere tho.

Reply:
I just realized we need to check the priority too, so it got a bit more complex.

I like getKeyExpirySec because I don't have to check every single time if it is milisec or sec.

Reply:
![image](https://github.com/guildxyz/guild-queues/assets/56923685/2b1f98bf-32a6-4c6e-9252-080c462f849e)
and it's shorter :harold:

Reply:
I just merge it now to see the results sooner, but we can come back to this naming thing later.